functional-test/src/test/groovy/org/rundeck/tests/functional/api/project/ConfigSpec.groovy

```diff
@@ -0,0 +1,485 @@
+package org.rundeck.tests.functional.api.project
+
+import org.rundeck.tests.functional.api.ResponseModels.ConfigProperty
+import org.rundeck.tests.functional.api.ResponseModels.Execution
+import org.rundeck.tests.functional.api.ResponseModels.ProjectCreateResponse
+import org.rundeck.util.annotations.APITest
+import org.rundeck.util.container.BaseContainer
+import org.testcontainers.shaded.com.fasterxml.jackson.databind.ObjectMapper
+import org.testcontainers.shaded.org.yaml.snakeyaml.Yaml
+
+@APITest
+class ConfigSpec extends BaseContainer{
+
+    def "test GET /api/14/project/name/config/key"(){
+        given:
+        def client = getClient()
+        client.apiVersion = 14 // as the original test
```

Comment:
use the default api version unless the test is specifically for other API version